internal/client/config_test.go

```diff
@@ -0,0 +1,92 @@
+package client
+
+import (
+	"crypto/x509"
+	"net/http"
+	"path/filepath"
+	"testing"
+
+	apiclient "github.com/flightctl/flightctl/internal/api/client"
+	"github.com/flightctl/flightctl/internal/crypto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	caCertValidityDays          = 365 * 10
+	serverCertValidityDays      = 365 * 1
+	clientBootStrapValidityDays = 365 * 1
+	signerCertName              = "ca"
+	clientBootstrapCertName     = "client-enrollment"
+)
+
+func TestClientConfig(t *testing.T) {
+	assert := assert.New(t)
+	require := require.New(t)
```

Comment:
nit: just need require here its not common to use both but I see why you did. require has all of the assertion methods that assert does.

Reply:
Changed that to use `require` only. Reasons I sprinkled in `assert` were that (unlike `require`) it doesn't immediately terminate the test, so provides more useful feedback in the same run, and it can be used in conditionals.